nc-news/src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from "react";
+import ArticlesCard from "./ArticlesCard";
+import { getAllArticles } from "../api";
+
+const ArticlesList = () => {
+  const [articles, setArticles] = useState([]);
+  useEffect(() => {
+    getAllArticles().then((articlesFromApi) => {
+      setArticles(articlesFromApi);
+    });
+  }, []);
```

Comment:
We should take into account that the data might take a long time to be fetched and we would want to have a loading screen for user feedback